src/test/bootstrap/.cliversion

```diff
@@ -1 +1 @@
-1.0.0-preview3-003886
+2.0.0-preview1-005899
```

Comment:
I thought we can get rid of this "boostrap" thing now?  Is this a follow up item?